doc/source/catalog_generation.rst

```diff
@@ -0,0 +1,361 @@
+.. _catalog_generation:
+
+==================
+Catalog Generation
+==================
+
+Point (Aperture) Photometric Catalog Generation
+================================================
+
+1: Source Detection
+-------------------
+
+1.1: Important Clarifications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+As described in the previous step, AstroDrizzle creates a single multi-filter detector-level drizzle-combined
+image for source identification and one or more detector/filter-level drizzle-combined images (depending on
+which filters were used in the dataset) for photometry. The same set of sources identified in the
+multi-filter detection image is used to measure photometry for each filter. We use method to maximize the
+signal across all available wavelengths at the source detection stage, thus providing photometry with the
+best quality source list across all available input filters.
+
+It should also be stressed here that the point and segment photometry source list generation algorithms
+identify source catalogs independently of each other and DO NOT use a shared common source catalog for
+photometry.
+
+1.2: Preliminaries
+^^^^^^^^^^^^^^^^^^^^
+
+1.2.1: Generation of the Bad Pixel Mask
+""""""""""""""""""""""""""""""""""""""""""""""""
+Before any source identification takes place, we created a bad pixel mask to identify regions of the
+detection image where signal quality is known to be degraded. These are areas near the edge of the image,
+areas with little to no input image contribution, and areas that contain saturated pixels. To minimize the
+impact of these regions on source identification and subsequent photometric measurements, the regions flagged
+in this bad pixel mask are iteratively “grown” for 10 steps using the `ndimage.binary_dilation <https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.binary_dilation.html>`_ scipy tool.
+Pixels in the immediate vicinity of a given masked region may also be impacted to some degree. As we cannot
+be fully certain that these pixels are or are not impacted, or to the degree of the impact, they are all
+flagged.
+
+1.2.2: Detection Image Background Subtraction
+""""""""""""""""""""""""""""""""""""""""""""""""
+To ensure optimal source detection, the multi-filter detection image is background-subtracted. We computed a
+2-dimensional background image using the `photutils.background.Background2d <https://photutils.readthedocs.io/en/stable/api/photutils.background.Background2D.html>`_ Astropy tool. This algorithm uses
+sigma-clipped statistics to determine background and RMS values across the image. An initial low-resolution
+estimate of the background is performed by computing sigma-clipped median values in 27x27 pixel boxes across
+the image. This low-resolution background image is then median-filtered using a 3x3 pixel sample window to
+correct for local small-scale overestimates and/or underestimates.
+
```

Comment:
For the 27x27 and 3x3, it should be noted these are configurable values.  Our catalogs use these values deeming them to be the best for the general situation, but a user could twiddle the values to optimize for their own data.

Reply:
Issue resolved in commit [203ed51](https://github.com/spacetelescope/drizzlepac/pull/760/commits/203ed519b49749f7ad519df95710174ab98faffe)